pkg/pillar/types/zedroutertypes.go

```diff
@@ -626,7 +626,9 @@ func GetExistingInterfaceList(globalStatus DeviceNetworkStatus) []string {
 	return ifs
 }
 
+// IsPort
 // Check if an interface/adapter name is a port owned by zedrouter
+//  Checks if the name IfName matches the given port.
```

Comment:
didn't parse for me; I had to read the code. I think what you meant to say
// Compares both the Name and the IfName against the port argument, and matches on either.